ccloudvm.go

```diff
@@ -140,12 +140,13 @@ func (d *drives) Set(value string) error {
 	return nil
 }
 
-func vmFlags(fs *flag.FlagSet, memGB, CPUs *int, m *mounts, p *ports, d *drives) {
+func vmFlags(fs *flag.FlagSet, memGB, CPUs *int, m *mounts, p *ports, d *drives, qemuport *uint) {
 	fs.IntVar(memGB, "mem", *memGB, "Gigabytes of RAM allocated to VM")
 	fs.IntVar(CPUs, "cpus", *CPUs, "VCPUs assigned to VM")
 	fs.Var(m, "mount", "directory to mount in guest VM via 9p. Format is tag,security_model,path")
 	fs.Var(d, "drive", "Host accessible resource to appear as block device in guest VM.  Format is path,format[,option]*")
 	fs.Var(p, "port", "port mapping. Format is host_port-guest_port, e.g., -port 10022-22")
+	fs.UintVar(qemuport, "qemuport", *qemuport, "Port to follow qemu logs of the guest machine, eg., -q 9999")
```

Comment:
eg., -q 9999

should be

e.g., -qemuport 9999

Reply:
*facepalm*. WIll fix it.